main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -2,25 +2,22 @@
 
 Here, we overview the fundamental concepts involved with building orchestration smart contracts.
 
-
 ### Interchain Account (ICA)
 
-[Interchain Accounts](/glossary/#interchain-account-ica) (ICAs) are an IBC feature used in Agoric’s orchestration API. They enable an Agoric smart contract to control an account on another blockchain within the Cosmos ecosystem, facilitated by Agoric [Orchestration](#orchestration) API. This feature leverages the [Inter-Blockchain Communication (IBC)](#ibc) protocol to facilitate interactions and transactions across different blockchains seamlessly.
-
+[Interchain Accounts](/glossary/#intercahin-account-ica) (ICAs) are an IBC feature used in Agoric’s Orchestration API. They enable an Agoric smart contract to control an account on another blockchain within the Cosmos ecosystem, facilitated by Agoric [Orchestration](/glossary/#orchestration) API. This feature leverages the [Inter-Blockchain Communication (IBC)](/glossary/#ibc) protocol to facilitate interactions and transactions across different blockchains seamlessly.
```

Comment:
intercahin -> interchain

Reply:
Fixed. Thanks for catching this. Noticed that the link is working fine but there was a typo in the Glossary for the InterChain Account section.